src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  padding: 0 20px;
+
+  @include tablet {
+    padding: 0 27px;
+  }
+
+  @include desktop {
+    padding: 0 50px;
+    margin: auto;
+  }
+
+  @include big-desktop {
+    padding: 0 130px;
+    margin: auto;
+  }
+}
+
+html {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  line-height: 150%;
+  font-size: 16px;
+  scroll-behavior: smooth;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.title {
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 150%;
+  letter-spacing: 1.2px;
+  text-transform: uppercase;
+  margin-bottom: 10px;
+  color: $text-minor-color;
+
+  @include desktop {
+    line-height: 110%;
+    letter-spacing: 1.8px;
+    font-size: 36px;
+  }
+}
+
+footer {
+  background-color: $footer-background;
+}
+
+.no-scroll {
```

Comment:
The '.no-scroll' class seems to be a utility class. Consider moving it to a dedicated utilities file if you have one.